<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>library</class>
 <widget class="QDialog" name="library">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kütüphane</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/Window/image/icon-window.png</normaloff>:/Window/image/icon-window.png</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>361</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>My Games</string>
   </property>
   <widget class="QTableWidget" name="tWMyGames">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>20</y>
      <width>351</width>
      <height>171</height>
     </rect>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <column>
     <property name="text">
      <string>Oyun Id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Oyun İsmi</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Kategori</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnDelete">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>350</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(195, 0, 0);</string>
   </property>
   <property name="text">
    <string>Oyun Sil</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUpdate">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(195, 0, 0);</string>
   </property>
   <property name="text">
    <string>Güncelle</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">color: rgb(195, 0, 0);</string>
      </property>
      <property name="text">
       <string>Oyun Adı</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="leSearch"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>70</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QToolButton" name="toolbtnleft">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/left/image/left.png</normaloff>:/left/image/left.png</iconset>
      </property>
      <property name="arrowType">
       <enum>Qt::NoArrow</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="toolbtnright">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/right/image/right.png</normaloff>:/right/image/right.png</iconset>
      </property>
      <property name="arrowType">
       <enum>Qt::NoArrow</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="gameName">
        <property name="styleSheet">
         <string notr="true">color: rgb(195, 0, 0);</string>
        </property>
        <property name="text">
         <string>Oyun İsmi</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="category">
        <property name="styleSheet">
         <string notr="true">color: rgb(195, 0, 0);</string>
        </property>
        <property name="text">
         <string>Kategori</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="leGameName">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leCategory"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnAdd">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(195, 0, 0);</string>
   </property>
   <property name="text">
    <string>Oyun Ekle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNew">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(195, 0, 0);</string>
   </property>
   <property name="text">
    <string>Yeni</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
